src/clj/puppetlabs/puppetserver/certificate_authority.clj

```diff
@@ -1264,7 +1294,7 @@
                                             (create-agent-extensions
                                              csr
                                              cacert))]
-    (log/info (i18n/trs "Signed certificate request for {0}" subject))
+    (apply report-successful-cert-signing report-activity-or-nil (generate-cert-message-from-request request subject))
```

Comment:
I think apply isn't super idiomatic in our code bases and moving the `generate-cert-message-from-request` into the let block above and pulling out the returned values, eg:
```clj
(let [...snip...
      [msg signee certname ip] (generate-cert-message-from-request)]
  (report-successful-cert-signing report-activit-or-nil msg signee certname ip)
```
but both approaches are correct and this might just be a me preference.